kolibri/core/assets/src/core-app/index.js

```diff
@@ -71,6 +77,26 @@ heartbeat.startPolling();
 
 i18nSetup().then(coreApp.ready);
 
+// these shall be responsibe for catching runtime errors
+Vue.config.errorHandler = function (err, vm) {
+  logging.error(`Unexpected Error: ${err}`);
+  const error = new VueErrorReport(err, vm);
+  ErrorReportResource.report(error);
+};
+
+window.addEventListener('error', e => {
+  logging.error(`Unexpected Error: ${e.error}`);
+  const error = new JavascriptErrorReport(e);
+  ErrorReportResource.report(error);
+});
+
+window.addEventListener('unhandledrejection', event => {
+  event.preventDefault();
+  logging.error(`Unhandled Rejection: ${event.reason}`);
```

Comment:
I know the `unhandledrejection` listener will prevent default logging of the error, so in regards to that and the other logging statements, I'm concerned whether these are suppressing necessary log information, i.e. a stack trace, that developers would need? If `logging.error` outputs a stack trace, that may not be the same trace as the error itself.